README.md

```diff
@@ -198,14 +198,29 @@ pip install pyfstat[chainconsumer,pycuda,style]
 ```
 This command accepts the "development mode" tag `-e`.
 
-* If you prefer to make your own LALSuite installation
+### Using LALSuite built from source
+
+If you prefer to make your own LALSuite installation
 [from source](https://git.ligo.org/lscsoft/lalsuite/),
 make sure it is **swig-enabled** and contains at least the `lalpulsar` and `lalapps` packages.
-A minimal configuration line to use would be e.g.:
+
+Following the instructions from their README,
+at the step where it tells you to run `./configure`,
+an example minimal line would be e.g.:
+```
+./configure --prefix=${HOME}/lalsuite-install --disable-all-lal --enable-lalpulsar --enable-lalapps --enable-swig --enable-swig-python --disable-swig-octave
```

Comment:
Ah, you're right. I added `--disable-swig-octave` explicitly because @itsluanayall had problems without it, but indeed just `--enable-swig-python` without `--enable-swig` should be equivalent.